keps/sig-node/xxx-dynamic-resource-allocation/README.md

```diff
@@ -175,11 +182,11 @@ resources for containers. Later, support for storage and discrete,
 countable per-node extended resources was added. The device plugin
 interface then made such local resources available to containers. But
 for many newer devices, this approach and the Kubernetes API for
-requesting these custom resources are too limited. This KEP addresses
-the following limitations:
+requesting these custom resources is too limited. This KEP addresses
+limitations of the current approach for the following use cases:
 
 - *Device initialization*: When starting a workload I’d like to have
-  the device reconfigured or reprogrammed by the orchestration. For
+  the device reconfigured or reprogrammed at the orchestration. For
```

Comment:
Ah, now I understand this better. I think maybe "during" would be more appropriate, as in "during orchestration" (without the word "the").

Reply:
Changed.